src/Node/Crypto/Hash.purs

```diff
@@ -33,12 +35,24 @@ hex
   -> Effect String
 hex alg str = hash alg str Hex
 
+hexBuffer
+  :: Algorithm
+  -> Buffer
+  -> Effect String
+hexBuffer alg buf = hashBuffer alg buf Hex
+
 base64
   :: Algorithm
   -> String
   -> Effect String
 base64 alg str = hash alg str Base64
 
+base64Buffer
+  :: Algorithm
+  -> Buffer
+  -> Effect String
+base64Buffer alg buf = hashBuffer alg buf Base64
```

Comment:
This is another version shorthand function of `base64`, so i want you to rename `base64Buffer` to `base64'`.